canopen/emcy.py

```diff
@@ -22,18 +22,19 @@ def __init__(self):
 
     def on_emcy(self, can_id, data, timestamp):
         code, register, data = self.EMCY_STRUCT.unpack(data)
-        if code & 0xFF == 0:
+        entry = EmcyError(code, register, data, timestamp)
+
+        with self.emcy_received:
+             self.log.append(entry)
+             self.active.append(entry)
+             self.emcy_received.notify_all()
+
+        if code & 0xFFFF == 0:
             # Error reset
             self.active = []
             for callback in self.callbacks:
                 callback(None)
```

Comment:
I'm thinking it may be better skip using `None` as a representation for reset and just use the same emcy object regardless if it is a reset or an actual error. Then you can get timestamp for the reset and so on. What do you think? Then the documentation would have to be changed slightly as well.